packages/pages/src/dev/server/middleware/serverRenderSlugRoute.ts

```diff
@@ -69,7 +69,7 @@ export const serverRenderSlugRoute =
         locale,
         document,
       });
-      sendAppHTML(res, templateModuleInternal, props, vite, url.pathname);
+      sendAppHTML(res, templateModuleInternal, props, vite, `/${slug}`);
```

Comment:
Does the leading slash in the slug matter? It looks like we didn't have it before but now have one.

Reply:
Through my testing, it seems like we actually did have it before. The `url.pathname` included a leading slash (`url.pathname.substring(1)` does not).